mimic/imimic.py

```diff
@@ -46,7 +70,51 @@ def catalog_entries(tenant_id):  # pragma:nocover
 
     def uri_for_service(region, service_id):  # pragma:nocover
         """
-        Get the URI for the service in a given region
+        Get the URI for the service in a given region.
+        """
+
+    def list_tenant_endpoints(tenant_id):  # pragma:nocover
```

Comment:
The purpose of an `Interface` is clear, explicit documentation of a system boundary, so `:param:` and `:return:` documentation is particularly important here; please add `:param:` and `:return:` for every one of these.